cmd/syndi/main.go

```diff
@@ -35,15 +42,18 @@ func main() {
 		log.Panic(err)
 	}
 
-	im := importer.NewImporter(db, cfg)
-	err = im.DisableFK()
-	if err != nil {
-		log.Panic(err)
-	}
-	defer im.EnableFK()
-
-	err = im.Import()
-	if err != nil {
-		log.Panic(err)
+	// import things
+	for _, tableDef := range tableDefinitions {
+		im := importer.NewImporter(db, tableDef)
```

Comment:
Nice to have:
Move contents of the for loop in a function. It will be stupidly easy to parallelize it afterwards by just prepending `go` to
```go
	for _, tableDef := range tableDefinitions {
		go importSingleTable(db, tableDef)
	}
```

Reply:
it's more complicated than that, `db` cannot be shared...